SR2021/2021-01-28-discord-mentoring.md

```diff
@@ -0,0 +1,29 @@
+---
+to: Teams who didn't attend session 1
+via: discord
+---
+
+Hey team $TLA,
+
+We noticed you didn't make it to the first league session. We wanted to reach
+out to see if there was any more support we could provide you, or questions we
+could answer?
+
+There's usually lots of volunteers around in Discord to help out if you need
+anything, whether it's help programming your robot, organising your team or you
```

Comment:
Worth also mentioning the docs in here?

Reply:
I wondered about that, but these messages are already quite long. If there's a way to fit it in which doesn't feel too contrived nor add much length I'm up for doing it, but it felt better to focus on getting the mentoring message across well rather than split foci.